### ISSUE TITLE
Enable the UI to run arbitrary AddOns 

### ISSUE THREAD

**Issue by jwmatthews** [konveyor/enhancements#99]
We would like to surface the below functionality in a Konveyor release, which supports allowing arbitrary AddOns to be invoked from the UI.

https://github.com/konveyor/tackle2-ui/tree/feature-addon
https://github.com/konveyor/tackle2-ui/pull/544

We should consider if we want to keep this functionality behind a feature flag or if we are comfortable enabling by default.

**Comment by shawn-hurley** [comment]
IF we have the ability to add arbitrary AddOns as an always on feature, then we should have a way to use them from the UI IMO.

**Comment by jmle** [comment]
@jwmatthews @shawn-hurley is this still relevant?

**Comment by shawn-hurley** [comment]
I don't think so. I don't know if there is a way to in the task manager or something else that would allow me to run an addon that isn't an analyzer addon @sjd78 @jortel. Am I wrong?

**Comment by sjd78** [comment]
> I don't think so. I don't know if there is a way to in the task manager or something else that would allow me to run an addon that isn't an analyzer addon [@sjd78](https://github.com/sjd78) [@jortel](https://github.com/jortel). Am I wrong?

Other addons are called for platform awareness and asset generation, but those are in the context of UI wizards.

It would be rather straight forward to create an admin page to allow creation of an arbitrary task (that calls an addon) and submit it.  Basically a simple UI on top of the task API.  Not sure if it is really all that useful compared to just using curl.
